demos/wandb/settings

```diff
@@ -0,0 +1,4 @@
+[default]
```

Comment:
Is this file needed?

Reply:
No it was some how added my bad i'll clean it up.